### DISCUSSION TITLE
Welcome to ha-renac Discussions!

### DISCUSSION THREAD

**Discussion by gastush** [gastush/ha-renac#17] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by datshi001** [comment]
Good idea to have a discussions tab..........
I'm Datshi & I'm the owner of a 3-phase Renac Inverter - particularly pleased to have come across the @gastush integration which is now the main driver of my dashboard along with my (Altherma) heat pump.

**Comment by datshi001** [comment]
Hi @gastush - I've been using version 0.0.8 for a couple of days now and it seems fine - although yesterday it returned a single 'rogue' reading in the middle of the day from 'todays generated power' -  approx double what it should have been - i've no idea what could have caused this, it hadn't happened before - the readings either side were fine. I had been messing with the forecast integration (which failed miserably) so maybe it was something connected to that...........

![image](https://github.com/gastush/ha-renac/assets/155902554/3ffffc2c-a538-4638-878f-a274f91bb3c8)
